mod-orders-storage/schemas/cost.json

```diff
@@ -3,14 +3,35 @@
   "description": "purchase order line cost record",
   "type": "object",
   "properties": {
-    "listPrice": {
-      "description": "the per-item list price",
+    "listUnitPrice": {
+      "description": "The per-item list price",
```

Comment:
It would be nice to add a little more detailed description (ex. for "physical" and "other" items or something like that), just for maintainability, even (I didn't know until I read to the other comment in the PR)

Reply:
Done.
Only one note - even though now `listUnitPrice` is for physical resources or `Other` order format, it still works in combination with `quantityPhysical` which is a bit confusing...